main.go

```diff
@@ -6,18 +6,37 @@ import (
 	"github.com/giantswarm/microkit/command"
 	"github.com/giantswarm/microkit/logger"
 	microserver "github.com/giantswarm/microkit/server"
+	"github.com/spf13/viper"
 
+	k8sclient "github.com/giantswarm/cert-operator/client/k8s"
+	"github.com/giantswarm/cert-operator/flag"
 	"github.com/giantswarm/cert-operator/server"
 	"github.com/giantswarm/cert-operator/service"
 )
 
 var (
-	description string = "The cert-operator handles certificates for Kubernetes clusters running on Giantnetes."
-	gitCommit   string = "n/a"
-	name        string = "cert-operator"
-	source      string = "https://github.com/giantswarm/cert-operator"
+	description string     = "The cert-operator handles certificates for Kubernetes clusters running on Giantnetes."
+	f           *flag.Flag = flag.New()
+	gitCommit   string     = "n/a"
+	name        string     = "cert-operator"
+	source      string     = "https://github.com/giantswarm/cert-operator"
 )
 
+// Flags is the global flag structure used to apply certain configuration to it.
+// This is used to bundle configuration for the command, server and service
+// initialisation.
+var Flags = struct {
```

Comment:
This is not necessary anymore. Above you already create the new flag structure which is automatically set up. See line 19. 

```
f           *flag.Flag = flag.New()
```